using-timescaledb/visualizing-data.md

```diff
@@ -1,25 +1,20 @@
 # Visualizing data
 
-The time-series data stored in TimescaleDB is a natural fit for visualization.
-You can use SQL to manipulate data over time and funnel the results into the visualization
-tools or frameworks that you prefer.
-
+The time-series data stored in TimescaleDB can be easily displayed on graphs.
 TimescaleDB is compatible with visualization tools that work with PostgreSQL. This means that
 regardless of whether you are creating custom visualizations embedded in your applications or
 using off-the-shelf visualization tools to expose data across your business organization, you
 can choose from a wide selection of tools.
 
 ## Grafana [](grafana)
 
-Grafana is an open source visualization tool popular in the DevOps monitoring space,
-although it can also be used across the organization to visualize time series metrics.
+Grafana is an open-source visualization tool popular in the DevOps monitoring space,
+although it can also be used across the organization to visualize time-series metrics.
 Getting started with Grafana is simple. Download and install [Grafana][grafana-install].
 Then, add a new PostgreSQL data source that points to your TimescaleDB instance.
 Queries run through Grafana will continue to benefit from the performance improvements
-built into TimescaleDB.
-
-Once you have connected a data source, you can start building dashboards. Grafana
-allows you to write SQL queries to define dynamic dashboards.
+built into TimescaleDB. In fact, this data source was built by TimescaleDB engineers,
+and it is designed to take advantage of its time-series capabilities.
```

Comment:
"it" refers to two different things in this sentence.  Switch the second one to "the database's"?

Reply:
Done